CAN_Logger_with_Autobaud_and_Requests_no_CAN2_AES_CBC_SHA256/CAN_Logger_with_Autobaud_and_Requests_no_CAN2_AES_CBC_SHA256.ino

```diff
@@ -859,7 +835,28 @@ void myLongPressFunction(){
 
 
 void setup(void) {
+  Wire.begin(I2C_MASTER, 0x00, I2C_PINS_18_19, I2C_PULLUP_EXT, 100000);
+  if (atecc.begin() == true)
+  {
+    Serial.println("Successful wakeUp(). I2C connections are good.");
+  }
+  else
+  {
+    Serial.println("Device not found. Check wiring.");
+    while (1); // stall out forever
+  }
+  atecc.readConfigZone(false); // produces a serial number
+  
+  atecc.generatePublicKey(0,true);
+  
+  Serial.println("Starting CAN logger.");
 
+  // The server public key should already be installed in the ATECC. Why do we have to do this?
```

Comment:
according to: 
https://www.youtube.com/watch?v=SFpkqmwoEeg&t=23m57s
it's how the configuration of the 508a is currently set up